web/vapour-style.json

```diff
@@ -533,6 +557,85 @@
         }
       }
     },
+    {
+      "id": "stations-label",
+      "type": "symbol",
+      "metadata": {},
+      "source": "vapour_trail",
+      "source-layer": "stations",
+      "minzoom": 6,
```

Comment:
To enforce good mapping I would rather have 
* at low zooms, every bus station displayed as a point (the nodes, or the centroid for the areas)
* at medium zoom, we display areas (the areas, and maybe a circular buffer for the nodes)
* at high zoom, we may keep the geometries of the bus stations mapped as areas, but no label, because we now want to see the individual bus stops inside the bus station zone

What do you think guys ?